misc/test/azure_native_test.go

```diff
@@ -0,0 +1,70 @@
+//go:build AzureNative || all
+// +build AzureNative all
+
+package test
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/pulumi/examples/misc/test/definitions"
+	"github.com/pulumi/examples/misc/test/helpers"
+	"github.com/pulumi/pulumi/pkg/v3/testing/integration"
+)
+
+func TestAccAzureNative(t *testing.T) {
+	for _, examples := range definitions.AzureNativeTests {
```

Comment:
is there a reason you use `definitions.AzureNativeTests` here but `definitions.GetTestsForTag("azure")` for the azure tests instead of using the same technique for both?